jobs/JGLOBAL_LANDENS_ANALYSIS

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+export WIPE_DATA="NO"
+export DATA=${DATA:-${DATAROOT}/${RUN}landensanl_${cyc}}
```

Comment:
These are only needed if the job is using the working directory of a previous job. It appears this job is doing initialize/run/finalize all in one job.
```suggestion
```

Reply:
Removed as you suggested. Thanks.